rules-tests/Php81/Rector/FuncCall/NullToStrictStringFuncCallArgRector/Fixture/skip_existing_casts.php.inc

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Rector\Tests\Php81\Rector\FuncCall\NullToStrictStringFuncCallArgRector\Fixture;
+
+final class DemoFile
+{
+    public function run()
+    {
+        return $relpath
+            . '/' . rawurlencode((string) $this->Get('account_code') ?: $this->Get('customer_id'));
```

Comment:
the detected value is not cast, but ternary, this may need to be changed as:

```diff
-    . '/' . rawurlencode((string) $this->Get('account_code') ?: $this->Get('customer_id'));
+    . '/' . rawurlencode((string) $this->Get('account_code') ?: (string) $this->Get('customer_id'));
```

Reply:
Perhaps the rule should have first wrapped the expression in parentheses

Reply:
https://github.com/rectorphp/rector-src/pull/7091